database/migrations/2021_07_27_191709_adds_label_measurement_type.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddsLabelMeasurementType extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('settings', function (Blueprint $table) {
+            $table->string('labels_measurement_type')->after('labels_per_page')->default('in');
+            $table->string('old_labels_measurement_type')->after('labels_measurement_type');
```

Comment:
I think you're trying to re-add the rest of the settings table here.

Reply:
These adjustments are actually necessary changes to the precision of the decimal being stored. the values blow up  if I dont adjust the precision here.